### PR TITLE
Vape Fix, no more Admin Spam edition. 

### PR DESCRIPTION
**Pull request by Cuboos**
OKAY... **HOPEFULLY** I didn't fuck up this time...

This should fix the Admin Message spam caused by the smoke spread. 

@optimumtact if i fuck up this time, kill me.

### CODE REVIEW

File: html/changelogs/vapeFix.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Removed the admin message spam from the smoke_spread proc. You may now vape in peace without the threat of a Blue Space Artillery"
```
Comment (Shadowlight213) [inline html/changelogs/vapeFix.yml]: You uh don't need to do this. You make the CL in the pr body.

**Review by Shadowlight213 — COMMENTED** [review]
Don't need a changelog file

File: html/changelogs/vapeFix.yml (hunk 1 above)
Comment (Jordie0608) [inline html/changelogs/vapeFix.yml]: It doesn't really matter, this still works the same.

**Comment by ExcessiveUseOfCobblestone** [comment]
+1

**Pull request merged**